extensions/vertx-http/deployment/src/test/resources/conf/application-local-static-resources.properties

```diff
@@ -0,0 +1,3 @@
+quarkus.http.local-static-resources.enabled=true
+quarkus.http.local-static-resources.path=/local-static
```

Comment:
@mathias82 Can you please rename it to something like `/public-resource` to make it clearer it is a mapping to the local folder, otherwise we have `/local-static` URI path segment and the local folder also named `/local-static`

Reply:
`public-resources` is better